data/en/test_section_summary.json

```diff
@@ -0,0 +1,282 @@
+                        "questions": [{
+                            "id": "over-16-question",
+                            "title": "Is the person above over 16?",
+                            "description": "",
+                            "type": "General",
+                            "answers": [{
+                                "q_code": "5",
+                                "id": "what-is-your-age",
+                                "type": "Radio",
+                                "label": "",
+                                "mandatory": false,
+                                "options": [{
+                                        "label": "Yes",
+                                        "value": "Yes"
+                                    },
+                                    {
+                                        "label": "No",
+                                        "value": "No"
+                                    }
+                                ]
+                            }]
+                        }]
+                    },
+                    {
+                        "type": "Question",
+                        "id": "repeating-block-2",
+                        "title": "{{answers.first_name[group_instance]}} {{answers.last_name[group_instance]}}",
+                        "questions": [{
+                            "id": "working-status-question",
+                            "title": "What is the working status of this person?",
+                            "description": "",
+                            "type": "General",
+                            "answers": [{
+                                "q_code": "6",
+                                "id": "what-is-your-working-state",
+                                "type": "Radio",
+                                "label": "",
+                                "mandatory": false,
+                                "options": [{
+                                        "label": "Full time student",
+                                        "value": "Full time student"
+                                    },
+                                    {
+                                        "label": "Employed",
+                                        "value": "Employed"
+                                    },
+                                    {
+                                        "label": "Self employed",
+                                        "value": "Self employed"
+                                    },
+                                    {
+                                        "label": "Unemployed",
+                                        "value": "Unemployed"
+                                    }
+                                ]
+                            }]
+                        }]
+                    }
+                ],
+                "id": "repeating-group",
```

Comment:
I'd take out repeating groups as the summary page isn't currently designed to cope with these and would detach from the work you're doing.